test/hotspot/jtreg/testlibrary_tests/ir_framework/tests/TestPrivilegedMode.java

```diff
@@ -32,6 +32,7 @@
 /*
  * @test
  * @requires vm.flagless
+ * @requires vm.debug == true
```

Comment:
I think the test can also fail when you have a build without compilers. Just to be sure, I would add the following:
```suggestion
 * @requires vm.debug == true & vm.compMode != "Xint" & vm.compiler2.enabled & vm.flagless
```

Reply:
Thanks for your advice. Does `vm.compiler2.enabled & vm.flagless` include the situation `vm.compMode != "Xint"`, I mean `vm.compMode != "Xint"` not needed when set requires `vm.compiler2.enabled & vm.flagless`

Reply:
Yes, I guess it's not needed.

Reply:
The code has been changed according your advice. Thanks.